Dynatrace/dynatrace-operator
# Shorten test dynakube name to pass validation

## Description

[In a recent PR](https://github.com/Dynatrace/dynatrace-operator/pull/2375) we added validation for the length of dynakube names, and as it turns out 1 of our e2e tests had a dynakube that was rejected because of it.

Also added some missing e2e make targets

## How can this be tested?
Check if the new e2e targets work. (only 2. one had problem with name length validation)
```
make test/e2e/cloudnative/codemodules-with-proxy
make test/e2e/cloudnative/codemodules-with-custom-ca
```

## Checklist
